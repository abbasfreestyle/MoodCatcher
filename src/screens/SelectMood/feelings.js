export const feelingsList = [
  { id: 1, name: 'Happy' },
  { id: 2, name: 'Excited' },
  { id: 3, name: 'Anxious' },
  { id: 4, name: 'Nervous' },
  { id: 5, name: 'Relaxed' }
];
