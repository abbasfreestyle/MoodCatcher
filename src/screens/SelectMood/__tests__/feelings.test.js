import { feelingsList } from '../feelings';

describe('Screens > Select Mood > Feelints List', () => {
  test('should match snapshot of the list of feelings provided', () => {
    expect(feelingsList).toMatchSnapshot();
  });
});
