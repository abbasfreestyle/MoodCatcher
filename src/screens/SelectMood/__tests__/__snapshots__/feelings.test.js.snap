// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Screens > Select Mood > Feelints List should match snapshot of the list of feelings provided 1`] = `
Array [
  Object {
    "id": "1",
    "name": "Happy",
  },
  Object {
    "id": "2",
    "name": "Excited",
  },
  Object {
    "id": "3",
    "name": "Anxious",
  },
  Object {
    "id": "4",
    "name": "Nervous",
  },
  Object {
    "id": "5",
    "name": "Relaxed",
  },
]
`;
